.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
I want to use the container cache, but this required to not use sudo command. As the project is php, the pip command does not behave as the python one to handle package installation.